cslabs-backend/Controllers/UserModulesController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using CSLabsBackend.Models;
+
+namespace CSLabsBackend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserModulesController : BaseController
+    {
+        public UserModulesController(DefaultContext defaultContext, IMapper mapper) : base(defaultContext, mapper)
+        {
+        }
+        // GET api/values
+        [HttpGet]
+        public IActionResult Get()
+        {
+            return Ok(DatabaseContext.UserModules.ToList());
+        }
+        
+        //Get api
+        [HttpGet("{id}")]
+        public UserModule Get(int id)
+        {
+            return this.DatabaseContext.UserModules.Find(id);
```

Comment:
This request should check if the current user is allowed to view the user module or not. If the user doesn't own that user_module, then they should receive a 403, forbidden status code. Use the `GetUser()` method to check the id's on the module vs the current user.